pyproject.toml

```diff
@@ -146,7 +146,23 @@ ignore_missing_imports = true
   # arguments, which don't work in 3.7, which causes mypy to bail out early if you have
   # numpy installed.
   [[tool.mypy.overrides]]
-  module = 'numpy'
+  module = [
```

Comment:
We need to update the comment above, otherwise LGTM

Reply:
Ping @tgamblin